Rewritten "ZHA exception and deviation handling" explanation for readability.

Gamester17 opened the pull request:
## Proposed change

Partially rewrote "ZHA exception and deviation handling" explanation for readability.

PS: By the way, I also think that this "ZHA exception and deviation handling" section should be moved down on this page to be under the troubleshooting section, but I did not do so in this PR.


## Type of change
- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
- Link to parent pull request in the codebase: Not applicable.
- Link to parent pull request in the Brands repository: Not applicable.
- This PR fixes or closes issue: Not applicable.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Gamester17 committed:
Rewritten "ZHA exception and deviation handling" 

Partially rewrote "ZHA exception and deviation handling" explanation for readability.

PS: By the way, I also think that this "ZHA exception and deviation handling" section should be moved down on this page to be under the troubleshooting section.

frenck reviewed (dismissed):
Thanks, @Gamester17 👍 

Happy Hacktoberfest 🎉 

(Just make the build pass so we can merge)

Gamester17 committed:
Spellchecking zha.markdown trying to fix lint text error

Spellcheck of the "ZHA exception and deviation handling" section in zha.markdown.

Gamester17 committed:
Resolve conflict between current zha.markdown and patch

Resolve conflict between current zha.markdown and patch

Gamester17 committed:
Fix URLs in in zha.markdown

Fix URLs in in zha.markdown

Gamester17 committed:
Second attempt to fix URLs in zha.markdown

Second attempt to fix URLs in zha.markdown

Gamester17 committed:
Trying again to resolve conflict between patch and current zha.markdown

Trying again to resolve conflict between patch and current zha.markdown

Gamester17 commented:
Can I please get a little help? I do not see the remaining conflict now so I do not know exactly what and how to resolve it?

Gamester17 commented:
Maybe I should have also gone ahead and moved that "ZHA exception and deviation handling" section to troubleshooting?

Personally, I think that it belongs under the troubleshooting section beneath "KNOWING WHICH DEVICES ARE SUPPORTED".

Gamester17 committed:
Merge branch 'current' into patch-17
